applications/openshift/integrity/crypto/machine_volume_encrypted/rule.yml

```diff
@@ -23,6 +23,7 @@ identifiers:
 
 references:
   pcidss: Req-3.4.1
+  srg: SRG-APP-000014-CTR-000035,SRG-APP-000014-CTR-000040,SRG-APP-000416-CTR-001015,SRG-APP-000514-CTR-001315,SRG-APP-000560-CTR-001340,SRG-APP-000610-CTR-001385,SRG-APP-000635-CTR-001405
```

Comment:
I'm not sure if this is correct. `SRG-APP-000014-CTR-000035` talks about TLS 1.2 which IMO is not related to this rule.
I think we should go over the draft STIG and check, for each line:

- is there a rule that implements that check and fix?
- if yes: tag that rule with the SRG ID, add that rule to the controls file
- if no: create a card to add such rule